### PR TITLE
feature(axisLables): add in axis labels

### PR DESCRIPTION
**Pull request by joshbDev**

### CODE REVIEW

File: tests/browser/spec/XAxisLabels.spec.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            <XAxisLabels position="top" distance={2} tickCount={5} />
+          </XYPlot>
+        </div>
+      </div>
+    );
+    const rendered = mount(tree).find(XAxisLabels);
+    const first = rendered.first();
+    // each tick is a g and there's a wrapper around all g's, hence the 10 and 6
+    expect(first.find("g")).to.have.length(10);
+    expect(rendered.at(1).find("g")).to.have.length(6);
```
Comment (krissalvador27) [inline tests/browser/spec/XAxisLabels.spec.js]: The 10 is for the number of ticks right? What's the 6 length in reference too?

File: tests/browser/spec/YAxisLabels.spec.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            <YAxisLabels position="right" tickCount={5} />
+          </XYPlot>
+        </div>
+      </div>
+    );
+    const rendered = mount(tree).find(YAxisLabels);
+    const first = rendered.first();
+    // each tick is a g and there's a wrapper around all g's, hence the 10 and 6
+    expect(first.find("g")).to.have.length(10);
+    expect(rendered.at(1).find("g")).to.have.length(6);
```
Comment (krissalvador27) [inline tests/browser/spec/YAxisLabels.spec.js]: same as above on XAxisLabels

File: tests/browser/spec/XAxisLabels.spec.js (hunk 1 above)
Comment (joshbDev) [reply tests/browser/spec/XAxisLabels.spec.js]: this test sets up two sets of ticker (line 37 and 38), so this is testing that the default is 9 and the manual setting is 5

Comment (krissalvador27) [reply tests/browser/spec/XAxisLabels.spec.js]: got it 👍

**Pull request merged**